keyboards/levinson/keymaps/iarumas/neo2-keys-on-quertz-de-latin1.h

```diff
@@ -0,0 +1,73 @@
+#ifndef NEO2_KEYS_ON_QUERTZ_DE_LATIN1_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?